codice/alliance
# CAL-71 Moved Metatype to appropriate directory and removed duplicate …

#### What does this PR do?
Fixes some refactoring errors in Alliance : 
Moves metatype from catalog-imaging-transformer to the renamed package imaging-transformer-nitf
Removes some duplicate classes in catalog-mpegts-stream that were not properly renamed
#### Who is reviewing it?
@dcruver 
@glenhein 
@jlcsmith 
@kcwire 
#### How should this be tested?
Build Alliance and verify that the metatype for the nitf imaging transformer works as intended and that FMV streaming still works as intended.
#### Any background context you want to provide?
Some packages were renamed and a couple of items were missed 
#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-71
#### Screenshots (if appropriate)
#### Checklist:

…classes lingering from refactoring
